app/helpers/alchemy/elements_helper.rb

```diff
@@ -159,7 +159,7 @@ def render_element(element, part = :view, options = {}, counter = 1)
         locals: options.delete(:locals) || {}
       }
 
-      element.store_page(@page) if part.to_sym == :view
+      element.store_page(@page) if part.to_sym == :view && @page.cache_page?
```

Comment:
I think now that the `if` condition is having 2 conditionals, we shouldn't use a postfix `if` here anymore.

It's harder to scan the rather long method.